src/utils/numbers.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Get a number in the given range of numbers based on the given percentage.
+ */
+export function lerp(from: number, to: number, percentage: number): number {
```

Comment:
Suggestion (blocking): From my understanding, the third parameter is not a [percentage](https://en.wikipedia.org/wiki/Percentage) – how about `t` (most common), `blend`, `factor`, or `weight`?